.travis.yml

```diff
@@ -2,12 +2,14 @@ language: php
 
 php:
   - 5.6
-  - 7.0
+  - 7.0.19
```

Comment:
Please don't do this. Also, php 5.6 is loaded at an older version too. Just only do dist trusty for hhvm.